modules/pages/client/components/FaqTechnology.component.js

```diff
@@ -42,7 +42,7 @@ export default function FaqTechnology() {
           limited software you can imagine that still works for hospitality. We
           simply haven&apos;t had time to build most of the features yet. Please
           be patient and be mindful about{' '}
-          <a href="http://www.productstrategymeanssayingno.com/">
+          <a href="https://www.productstrategymeanssayingno.com/">
```

Comment:
The https version hangs for me at the moment. Only the http link redirects to a meaningful website atm.